virttest/qemu_vm.py

```diff
@@ -3853,7 +3897,7 @@ def deactivate_nic(self, nic_index_or_name, wait=20):
             nic_eigenvalue = r'dev:\s+%s,\s+id\s+"%s"' % (nic.nic_model,
                                                           device_id)
             if not utils_misc.wait_for(lambda: not re.search(nic_eigenvalue,
-                                       self.monitor.info("qtree", debug=False)),
+                                                             self.monitor.info("qtree", debug=False)),
```

Comment:
This is a very long line... On some places you break lines even when not that necessary and here you keep such a long lines. It'd be appreciated to go through the style issues and address them semi-manually.